README.rst

```diff
@@ -192,6 +212,8 @@ Authors
 * `David Wihl`_
 * `Ben Karl`_
 
+.. _Installed Application Flow: https://developers.google.com/google-ads/api/docs/client-libs/dotnet/oauth-installed
```

Comment:
this is pointing to the .NET page. Without moving over the whole wiki, can we at least add the page for Python? Same comment for the following line.

Reply:
Thanks for catching this! I still need to add the Python docs to that page. Let's discuss if it makes sense to add just the sections linked here.

Reply:
This is updated now that the Python devsite section is published.